graphics/opencv3/Portfile

```diff
@@ -104,40 +118,62 @@ post-patch {
-                    -DOPENCV_3P_LIB_INSTALL_PATH=${prefix}/lib/${subport}/3rdparty \
-                    -DOPENCV_LIB_ARCHIVE_INSTALL_PATH=${prefix}/lib/${subport}/static \
-                    -DCMAKE_BINARY_DIR=${prefix}/libexec/${subport}/bin \
-                    -DCMAKE_INSTALL_LIBEXECDIR=${prefix}/libexec/${subport} \
-                    -DCMAKE_INSTALL_BINDIR=${prefix}/libexec/${subport}/bin \
-                    -DCMAKE_INSTALL_SBINDIR=${prefix}/libexec/${subport}/sbin \
-                    -DCMAKE_INSTALL_SHAREDIR=${prefix}/libexec/${subport}/share \
-                    -DCMAKE_INSTALL_INCLUDEDIR=${prefix}/include/${subport} \
-                    -DCMAKE_INSTALL_LIBDIR=${prefix}/lib/${subport} \
-                    -DCMAKE_INSTALL_NAME_DIR=${prefix}/lib/${subport} \
-                    -DCMAKE_INSTALL_RPATH=${prefix}/lib/${subport} \
+                    -DOPENCV_CONFIG_INSTALL_PATH=${opencv_install_cmake} \
+                    -DOPENCV_BIN_INSTALL_PATH=${opencv_install_prefix}/bin \
+                    -DOPENCV_SETUPVARS_INSTALL_PATH=${opencv_install_prefix}/scripts \
+                    -DOPENCV_SBIN_INSTALL_PATH=${opencv_install_prefix}/sbin \
+                    -DOPENCV_JAR_INSTALL_PATH=${opencv_install_prefix}/java/jar \
+                    -DOPENCV_JNI_INSTALL_PATH=${opencv_install_prefix}/java/jni \
+                    -DOPENCV_JNI_BIN_INSTALL_PATH=${opencv_install_prefix}/java/jni \
+                    -DOPENCV_TEST_INSTALL_PATH=${opencv_install_prefix}/test/bin \
+                    -DOPENCV_TEST_DATA_PATH=${opencv_install_prefix}/test/data \
+                    -DOPENCV_SAMPLES_BIN_INSTALL_PATH=${opencv_install_prefix}/share/samples/bin \
+                    -DOPENCV_SAMPLES_SRC_INSTALL_PATH=${opencv_install_prefix}/share/samples/src \
+                    -DOPENCV_OTHER_INSTALL_PATH=${opencv_install_prefix}/share/data \
+                    -DOPENCV_DOC_INSTALL_PATH=${opencv_install_prefix}/share/doc \
+                    -DOPENCV_LICENSES_INSTALL_PATH=${opencv_install_prefix}/share/license \
+                    -DOPENCV_INCLUDE_INSTALL_PATH=${opencv_install_include} \
+                    -DOPENCV_LIB_INSTALL_PATH=${opencv_install_lib} \
+                    -DOPENCV_3P_LIB_INSTALL_PATH=${opencv_install_lib}/3rdparty \
+                    -DOPENCV_LIB_ARCHIVE_INSTALL_PATH=${opencv_install_lib}/static \
+                    -DCMAKE_BINARY_DIR=${opencv_install_prefix}/bin \
+                    -DCMAKE_INSTALL_LIBEXECDIR=${opencv_install_prefix} \
+                    -DCMAKE_INSTALL_BINDIR=${opencv_install_prefix}/bin \
+                    -DCMAKE_INSTALL_SBINDIR=${opencv_install_prefix}/sbin \
+                    -DCMAKE_INSTALL_SHAREDIR=${opencv_install_prefix}/share \
+                    -DCMAKE_INSTALL_INCLUDEDIR=${opencv_install_include} \
+                    -DCMAKE_INSTALL_LIBDIR=${opencv_install_lib} \
+                    -DCMAKE_INSTALL_NAME_DIR=${opencv_install_lib} \
```

Comment:
Note to Reviewers:

The cmake portgroup does not expose an override for `CMAKE_INSTALL_NAME_DIR`, and it's hard-coded to `${cmake.install_prefix}/lib`. So we have no choice but to explicitly specify it ourselves.